genkit-tools/cli/src/commands/init/init-go.ts

```diff
@@ -36,48 +36,30 @@ interface ModelOption {
 /** Path to Genkit sample template. */
 const templatePath = '../../../config/main.go.template';
 
-const googleaiInitTmpl = `if err := googleai.Init(context.Background(), ""); err != nil {
-		log.Fatal(err)
-	}
-	for _, mname := range googleai.KnownModels() {
-		_, err := googleai.DefineModel(mname, nil)
-		if err != nil {
-			log.Fatal(err)
-		}
-	}`;
-
-const vertexaiInitTmpl = `if err := vertexai.Init(context.Background(), "" /* TODO: Set project ID. */, "us-central1"); err != nil {
-		log.Fatal(err)
-	}
-	for _, mname := range vertexai.KnownModels() {
-		_, err := vertexai.DefineModel(mname, nil)
-		if err != nil {
-			log.Fatal(err)
-		}
-	}`;
-
-const ollamaInitTmpl = `if err := ollama.Init(context.Background(), ollama.Config{ServerAddress: "http://127.0.0.1:11434", Models: []ollama.ModelDefinition{{Name: "gemma"}}}); err != nil {
-		log.Fatal(err)
-	}`;
+/** Ollama init call template. Keep indenting to for expected output. */
+const ollamaInit = `ollama.Init(context.Background(), ollama.Config{
```

Comment:
Do all these end up in `main`? If so, let's start main with 
```
ctx := context.Background()
```
and use `ctx` throughout.